app/AccountancyModule/EducationModule/presenters/ParticipantPresenter.php

```diff
@@ -138,6 +139,11 @@ public function actionExport(int $aid): void
     /** @return Participant[] */
     private function eventParticipants(): array
     {
-        return $this->queryBus->handle(new EducationParticipantListQuery($this->event->getId()));
+        return array_filter(
+            $this->queryBus->handle(new EducationParticipantListQuery($this->event->getId())),
```

Comment:
Proc nevraci uz vyfiltrované uvastniky to `EducationParticipantListQuery`?

Reply:
Já jsem nad tím přemýšlel, zda to nehodit přímo tam, ale přijde mi, že to není job té query - ta má vrátit SkautISová data o účastnících a to dělá, včetně sloupce `IsAccepted`. Dal by se okolo toho napsat nějaký wrapper, ale to mi už přišlo overkill...

Reply:
ja to nechci filtrovat na 5 místech, raději bych to dal jako parametr té query a defaultně bude pouze IsAccepted

Reply:
Parametr je dobrej :+1: